apps/docs/content/getting-started/overview/installation.mdx

```diff
@@ -0,0 +1,32 @@
><g fill="#fff" clip-path="url(#a)"><path d="M50 53.416c4.922 0 8.913-3.992 8.913-8.916s-3.99-8.916-8.913-8.916c-4.922 0-8.913 3.992-8.913 8.916s3.99 8.916 8.913 8.916Z"/><path fill-rule="evenodd" d="M7.389 39.004c-2.151 1.943-3.041 3.804-3.041 5.496 0 1.692.89 3.553 3.041 5.496 2.151 1.942 5.386 3.792 9.568 5.389 8.35 3.189 20.032 5.207 33.043 5.207s24.693-2.018 33.043-5.207c4.182-1.597 7.417-3.447 9.568-5.389 2.151-1.943 3.041-3.804 3.041-5.496 0-1.692-.89-3.553-3.041-5.496-2.151-1.942-5.386-3.792-9.568-5.389-8.35-3.189-20.032-5.207-33.043-5.207s-24.693 2.018-33.043 5.207c-4.182 1.597-7.417 3.447-9.568 5.389Zm8.017-9.452c8.96-3.422 21.192-5.493 34.594-5.493 13.402 0 25.634 2.07 34.594 5.493 4.472 1.708 8.241 3.795 10.93 6.224C98.214 38.204 100 41.148 100 44.5c0 3.352-1.787 6.296-4.476 8.724-2.689 2.429-6.458 4.516-10.93 6.224-8.96 3.422-21.192 5.493-34.594 5.493-13.402 0-25.634-2.07-34.594-5.493-4.472-1.708-8.241-3.795-10.93-6.224C1.786 50.796 0 47.852 0 44.5c0-3.352 1.787-6.296 4.476-8.724 2.689-2.429 6.458-4.516 10.93-6.224Z" clip-rule="evenodd"/><path fill-rule="evenodd" d="M33.453 4.838c-2.758-.892-4.814-.733-6.28.113-1.464.847-2.63 2.548-3.237 5.383-.606 2.835-.59 6.562.119 10.983 1.414 8.828 5.508 19.957 12.013 31.229 6.506 11.272 14.094 20.383 21.03 26.021 3.474 2.825 6.692 4.703 9.45 5.595 2.757.892 4.813.733 6.278-.114 1.465-.846 2.632-2.547 3.238-5.382.606-2.835.59-6.562-.119-10.983-1.414-8.828-5.508-19.957-12.013-31.229-6.506-11.272-14.094-20.383-21.03-26.021-3.474-2.825-6.692-4.703-9.45-5.595Zm12.191 2.22c7.443 6.05 15.352 15.61 22.053 27.221 6.701 11.61 11.024 23.242 12.541 32.715.757 4.73.835 9.038.077 12.582-.757 3.543-2.413 6.563-5.315 8.24-2.902 1.675-6.344 1.6-9.791.484-3.447-1.115-7.138-3.337-10.854-6.358-7.442-6.05-15.35-15.61-22.052-27.221-6.701-11.61-11.024-23.242-12.541-32.715-.757-4.73-.835-9.038-.077-12.582.757-3.543 2.413-6.563 5.315-8.24C27.902-.49 31.344-.415 34.791.7c3.447 1.115 7.138 3.337 10.854 6.357Z" clip-rule="evenodd"/><path fill-rule="evenodd" d="M76.064 10.334c-.606-2.835-1.773-4.536-3.238-5.383-1.465-.846-3.521-1.005-6.279-.113-2.757.892-5.975 2.77-9.45 5.595-6.935 5.638-14.523 14.75-21.029 26.021-6.505 11.272-10.6 22.4-12.013 31.229-.708 4.421-.725 8.148-.119 10.983.607 2.835 1.773 4.536 3.238 5.383 1.465.846 3.521 1.005 6.279.113 2.757-.892 5.975-2.77 9.45-5.595 6.935-5.638 14.523-14.75 21.029-26.021 6.505-11.272 10.6-22.4 12.013-31.229.708-4.421.725-8.148.119-10.983Zm4.174 11.672c-1.517 9.473-5.84 21.104-12.541 32.715-6.701 11.61-14.61 21.17-22.053 27.221-3.715 3.02-7.406 5.243-10.853 6.358-3.447 1.115-6.89 1.191-9.791-.485-2.902-1.676-4.558-4.696-5.315-8.24-.758-3.543-.68-7.852.077-12.58 1.517-9.474 5.84-21.105 12.541-32.716 6.701-11.61 14.61-21.17 22.053-27.222C58.07 4.038 61.761 1.816 65.209.7c3.447-1.115 6.89-1.191 9.791.485 2.902 1.676 4.558 4.696 5.315 8.24.758 3.543.68 7.852-.077 12.58Z" clip-rule="evenodd"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h100v89H0z"/></clipPath></defs></svg>
```

Comment:
I know this won't be shown to the user, but maybe we should put these svgs in a separate file and import them?
Just an idea since it's pretty large.